<launch>
	<include file = "$(find pointgrey_camera_driver)/launch/camera.launch">
		<arg name="calibrated" value ="1"/>
		<arg name="camera_serial" value= "rrl3"/>
	</include>
	<group ns="camera">
		<node name ="rectifier" pkg = "image_proc" type ="image_proc">
		</node>
	</group>	
	<node name="camgui" pkg="rqt_gui" type="rqt_gui"/>
	<node name ="H_Est" pkg="iros18_vision" type="iros18_vision_homography_est">
		<param name = "roi_l" value = "1"/>
		<param name = "roi_r" value = "1295"/>
		<param name = "roi_u" value = "1"/>
		<param name = "roi_b" value = "963"/>
	</node>
</launch>

